viam-cartographer.go

```diff
@@ -70,22 +71,13 @@ func init() {
 			)
 		},
 	})
-
-	config.RegisterServiceAttributeMapConverter(
```

Comment:
this moved to https://github.com/viamrobotics/viam-cartographer/pull/62/files/34ca5218fa1f054b1371fb8be31ae81f2ea44431#diff-93098c528cfadf9d4319fd6b3a1eb42ef437c6a805da14f423688a6944a4d912R54